tez-api/src/main/java/org/apache/tez/dag/api/TezConfiguration.java

```diff
@@ -883,6 +883,25 @@ public TezConfiguration(boolean loadDefaults) {
       + "dag.cleanup.on.completion";
   public static final boolean TEZ_AM_DAG_CLEANUP_ON_COMPLETION_DEFAULT = false;
 
+  /**
+   * Boolean value. Instructs AM to delete vertex shuffle data if a vertex and all its
+   * child vertices at a certain depth are completed.
+   */
+  @ConfigurationScope(Scope.AM)
```

Comment:
what about collapsing the two options into one, according to [Sid's comment](https://issues.apache.org/jira/browse/TEZ-3363?focusedCommentId=16219468&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16219468)

Reply:
ack. I will fix this in next revision.